src/data/blog/v-vvv-e-eee.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Value and effort and prioritisation'
+slug: 'v-vvv-e-eee'
+image: null
+featured: 0
+page: 0
+status: 'published'
+language: 'en_US'
+meta_title: null
+meta_description: null
+author_id: 5
+created_at: '2020-01-30T21:26:07.000Z'
+created_by: 5
+updated_at: '2020-01-30T09:59:00.000Z'
+updated_by: 1
+published_at: '2020-01-30T16:12:19.000Z'
+published_by: 5
+visibility: 'public'
+mobiledoc: null
+amp: null
+author_name: 'Harry Harrold'
+author_slug: 'harry-harrold'
+author_image: '//www.gravatar.com/avatar/febf032ab16fef166824f5172cd87393?s=250&d=mm&r=x'
+author_cover: null
+author_facebook: null
+author_twitter: null
+---
+
+Every sprint, Neontribe’s developers are faced with a stack of work, in an order. Every sprint, we start with work from the top of the stack, and carry on until the sprint finishes.
+
+We order this stack using Vs and Es; rough ideas of the Value and Effort of a piece of work. Normally, these pieces of work at least start as user stories; value we are delivering to a user of the software we’re creating. Every piece of work will start with one to three Vs and one to three Es.
+
+(V) means the team believes a user story is Valuable to the project. It is Valuable enough to include, and helps the project meet its objectives in the light of the user research. If a user story is without Value, it has no place in the software the project delivers.
+
+(VV) means the team believes a user story is Very Valuable to the project. It is more Valuable than a user story which is just Valuable.
+
+(VVV) means the team believes a user story is Very Very Valuable to the project. It is more Valuable than a user story which is simply Very Valuable.
+
+(VVVV) is not an acceptable value; it probably means there’s something epic there that needs to be shattered into smaller tasks.
+
+These assessments of Value are used to help prioritise user stories. The more Valuable a user story is, the more likely it is to be included in the software the project eventually delivers but it’s the product owner’s call. Value gives us a good first stab at prioritisation, which we then consider again when we know more about Effort.
+
+(E) means a user story will take some Effort. Any user story needs design thinking, coding, testing and some discussions to deliver. No user story, however seemingly trivial, is without effort to finish to a satisfactory level of quality. A developer who gives a story an E probably knows exactly how to do it and that it’ll be only a few lines of code. 
+
+(EE) means a user story will take Extra Effort. It's more work than a user story that only takes Effort, but less than one which takes Extreme effort. It’s likely a developer who gives a story an EE will have a pretty good idea how to do it but it's more than a few lines of code. 
```

Comment:
No reference to Extreme effort, the next one is Extra Extra Effort

Reply:
Extreme removed.